cktap/cli.py

```diff
@@ -655,12 +672,14 @@ def export_to_core(cvc, pretty, slot):
         label=""  # label will be set to slot number
     )
     descriptor_list = []
-    for _slot in range(card.active_slot+1):
-        if slot and _slot not in slot:
+    for be_slot in range(card.active_slot+1):
+        if be_slot > 9:
```

Comment:
this is actually a bug that is also present in master... but imo correct way to fix this is to not let `card.active_slot` (on simulator) to reach number 10 --> have not tested real card yet so not sure about difference there

active_slot is 10 when all slots are unsealed

Reply:
yeah lets make the emulator do what the card does (not sure what that is)

Reply:
tested with `1.0.0`